server/re/include/irods_re_plugin.hpp

```diff
@@ -641,48 +660,55 @@ namespace irods {
                 const std::string& _instance_name,
                 const std::string& _rt,
                 msParamArray_t*    _ms_params,
-                const std::string& _out_desc) {
-
-            for( auto& re_pack : this->re_mgr_.re_packs_ ) {
-                if( _instance_name == re_pack.instance_name_ ) {
-                    return re_pack.re_->exec_rule_text(
-                            re_pack.re_ctx_,
-                            _rt,
-                            _ms_params,
-                            _out_desc,
-                            callback(*this));
+                const std::string& _out_desc)
+        {
+            if (_instance_name.empty()) {
+                for (auto&& re_pack : this->re_mgr_.re_packs_) {
+                    const auto err = re_pack.re_->exec_rule_text(re_pack.re_ctx_, _rt, _ms_params, _out_desc, callback(*this));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'err' is assigned a value that is never used.](https://app.codacy.com/manual/trel/irods/pullRequest?prid=4927542)